.circleci/config.yml

```diff
@@ -355,6 +353,24 @@ jobs:
             docker push ${DOCKERHUB_REPO}:build_test
             docker push ${DOCKERHUB_REPO}:build_ui
             docker push ${DOCKERHUB_REPO}:latest
+  cirrus_deploy:
+    working_directory: ~/cirrus
+    machine:
+      image: ubuntu-2004:2023.04.2 # Ubuntu 20.04, Docker v20.10.24, Docker Compose v2.17.2
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - deploy:
+          name: Deploy to latest
+          command: |
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            make cirrus_build cirrus_test
+            docker tag cirrus:dev ${DOCKERHUB_CIRRUS_REPO}:cirrus_build
```

Comment:
@jaredlockhart we will be requiring a repo for cirrus to tag the latest, who can we reach out to do that?